zone/corpse.h

```diff
@@ -142,6 +147,9 @@ class Corpse : public Mob {
 	int32		player_kill_item; /* Determines if Player Kill Item */
 	uint32		corpse_db_id; /* Corpse Database ID (Player Corpse) */
 	uint32		char_id; /* Character ID */
+	uint32		consent_group_id = 0; /* consented group id */
+	uint32		consent_raid_id  = 0; /* consented raid id  */
```

Comment:
I know we have comments in other properties of the class, preferably we'd like to just have the variable or function describe itself instead of putting comments if we can